include/systems/diff_system.h

```diff
@@ -123,7 +123,8 @@ class DifferentiableSystem : public ImplicitSystem,
    */
   virtual void assembly (bool get_residual,
                          bool get_jacobian,
-                         bool apply_heterogeneous_constraints = false) libmesh_override = 0;
+                         bool apply_heterogeneous_constraints = false,
+			 bool apply_no_constraints = false) libmesh_override = 0;
```

Comment:
You should be able run contrib/bin/reindent.sh on the files you changed. What editor are you using that doesn't automatically align stuff like this?